payjoin/src/core/receive/v2/mod.rs

```diff
@@ -309,7 +311,11 @@ impl ReceiverBuilder {
     }
 
     pub fn with_expiry(self, expiry: Duration) -> Self {
```

Comment:
I figured this would be renamed to from_now but it's not a critical todo. Can be cleaned up when expiry/expiration is made to be consistent.